synapse/tests/test_lib_cell.py

```diff
@@ -1034,3 +1035,79 @@ async def _slowFakeBackup(self, name=None, wait=True):
             async with self.getTestCore(dirn=bkupdirn5) as core:
                 nodes = await core.nodes('test:str=ssl')
                 self.len(1, nodes)
+
+    async def test_cell_procpool(self):
+
+        ctor = s_cell.Cell
+
+        def _hasSlabFd(slabname, pid):
+            path = f'/proc/{pid}/fd/'
```

Comment:
A test link this should be flagged as linux only

Reply:
this test has been removed